transform/snowflake-dbt/models/hightouch/freemium/freemium_account_update.sql

```diff
@@ -0,0 +1,21 @@
+{{config({
+    "schema": "hightouch",
```

Comment:
@munish7771 , what is the difference from  `freemium_account.sql'? Also, why is `freemium_account_cancelled.sql` empty?

Reply:
instead of upsert, we do updates and inserts in different syncs. Accounts that already exist in salesforce will be updated.

Reply:
cancelled subscriptions is a part of another task. Accidently added file in this pr, removing it.

Reply:
> instead of upsert, we do updates and inserts in different syncs. Accounts that already exist in salesforce will be updated.

didn't see any difference code-wise, therefore the q.

Reply:
theres a diff in filter, 
` WHERE account.id IS NOT NULL AND account_domain_mapping.accountid IS NOT NULL` for account_update